services-gateway-client-transport/src/main/java/io/scalecube/services/gateway/transport/GatewayClientSettings.java

```diff
@@ -9,13 +9,15 @@
 
   private static final String DEFAULT_HOST = "localhost";
   private static final String DEFAULT_CONTENT_TYPE = "application/json";
+  private static final long DEFAULT_KEEPALIVE_INTERVAL = 30_000;
 
   private final String host;
   private final int port;
   private final String contentType;
   private final boolean followRedirect;
   private final SslProvider sslProvider;
   private final ServiceClientErrorMapper errorMapper;
+  private final long keepaliveIntervalMs;
```

Comment:
Could you make it as `Duration ` and [here](https://github.com/scalecube/scalecube-gateway/pull/95/files#diff-d4a33ef8f33409a906bd691f63f4d8d7R152-R155) convert to millis. In this case variable would be called just `keepaliveInterval`, and you wouldn't need to add `Ms` all over the places.